P0901.bs

```diff
@@ -564,6 +564,61 @@ void operator delete[](void* ptr, std::size_t size, std::align_val_t alignment)
 </table>
 </blockquote>
 
+# Implementation, Deployment and Performance Experience # {#impldepl}
+
+## Implementability
+
+The fundamental size feedback logic is a shipping part of TCMalloc ([[MallocExtension]]),
+as a set of functions such as:
+<pre>
+tcmalloc::sized_ptr_t tcmalloc_size_returning_operator_new(std::size_t size);
+</pre>
+
+We implemented the new `::operator new` overload in Clang. This
+presented no noteworthy complication or obstacle. An `::operator new`
+whose return type is not `void*` is novel, but entirely implementable.
+
+Building a few large applications from Google's codebase with the
+modified version of Clang showed nothing unexpected and continued
+working.
+
+## Deployment
+
+We modified the libc++ standard library implementation with minimal effort to
+use that function. (The implementation has a convenient
+<a href="https://github.com/llvm/llvm-project/blob/ed8dae9a438dd8b76a9760506fb7333ce7057d96/libcxx/include/__memory/allocate_at_least.h#L55">central point</a>
+at which the underlying allocation function is called.)
+We were able to build a number of significant production binaries of the Google
+codebase without problems.
+
+As above, building a few large applications from Google's codebase
+succeeded without incident.
+
+## Performance
+
+At Google, we modified libc++'s `basic_string` to use Clocks size-returning allocation
```

Comment:
Clocks?  Did you mean TCMalloc?

Reply:
What the? I don't even. No idea how this happened, maybe I accidentally allowed the spellchecker to change things? Obviously this is nonsense, sorry!